docs/installation/mail.md

```diff
@@ -8,7 +8,25 @@ For this, you will require an SMTP server. If you don't have one of these, your
 * [Postmark](https://postmarkapp.com/)
 * [Mailgun](https://signup.mailgun.com/new/signup) (the [free plan](https://www.mailgun.com/pricing) should be sufficient)
 * [Amazon Simple Email Service](https://aws.amazon.com/ses/)
+* [Sendgrid](https://sendgrid.com)
 
+## Use SMTP with Monica
+
+The generic way to send emails with Monica is to provide a SMTP server, each one of the services mentionned above can provide you SMTP settings. While Amazon SES is a little bit custom, see bellow, here the configuration for a standard SMTP configuration.
+
+You need to add few EVAR in your configuration (working in generic installation and Docker):
```

Comment:
Are EVAR the acronym for environment variables? If so we should avoid acronyms in the documentation if possible.

Reply:
fair! updated!